sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -155,6 +164,12 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
       settingKey[Map[String, util.JSON]]("Additional option to include in the generated 'package.json'")
 
     /**
+      * What [[scalajsbundler.BundlingMode]] should we use?
+      */
+    val webpackBundlingMode: SettingKey[BundlingMode] =
+      settingKey[BundlingMode]("What BundlingMode should we use?")
```

Comment:
I would not use an interrogative form here. Could you rephrase it in a declarative form?

Reply:
Also, you should mention in the scaladoc what the default value is.